packages/http-client/src/typekit/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./kits/index.js";
```

Comment:
@bterlson should this live in an "experimental" folder and namespace like it does for compiler and http?

Reply:
Updated to experimental folder.